docs/src/main/asciidoc/building-my-first-extension.adoc

```diff
@@ -383,6 +389,103 @@ The key points are:
 <3> We add  the `quarkus-bootstrap-maven-plugin` to generate the Quarkus extension descriptor included into the runtime artifact which links it with the corresponding deployment artifact.
 <4> We add  the `quarkus-extension-processor` to the compiler annotation processors.
 
+=== Gradle setup
+
+Quarkus does not provide any way to initialize a gradle project for extension yet.
```

Comment:
```suggestion
Quarkus does not provide any way to initialize a Gradle project for extension yet.
```

Reply:
There is a way to do that using the CLI. `quarkus create --gradle`. You could reference the CLI page.